htdocs/install/mysql/data/llx_accounting_account_fr.sql

```diff
@@ -1598,4 +1599,1128 @@ INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, pcg
 INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, pcg_subtype, account_number, account_parent, label, active) VALUES (__ENTITY__, 7112,'PCG18-ASSOC','INCOME',      'XXXXXX',   '87', '7208', 'Contributions volontaires en nature', 1);
 INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, pcg_subtype, account_number, account_parent, label, active) VALUES (__ENTITY__, 7113,'PCG18-ASSOC','INCOME',      'XXXXXX',  '870', '7112', 'Dons en nature', 1);
 INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, pcg_subtype, account_number, account_parent, label, active) VALUES (__ENTITY__, 7114,'PCG18-ASSOC','INCOME',      'XXXXXX',  '871', '7112', 'Prestations en nature', 1);
-INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, pcg_subtype, account_number, account_parent, label, active) VALUES (__ENTITY__, 7115,'PCG18-ASSOC','INCOME',      'XXXXXX',  '875', '7112', 'Bénévolat', 1);
\ No newline at end of file
+INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, pcg_subtype, account_number, account_parent, label, active) VALUES (__ENTITY__, 7115,'PCG18-ASSOC','INCOME',      'XXXXXX',  '875', '7112', 'Bénévolat', 1);
+
+--
+-- Descriptif des plans comptables FR PCGA14-DEV
+--
+
+INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, pcg_subtype, account_number, account_parent, label, active) VALUES (__ENTITY__, 8000,'PCGAFR14-DEV','CAPIT',		'XXXXXX',		'1',   '0', 'Comptes de capitaux', 1);
+INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, pcg_subtype, account_number, account_parent, label, active) VALUES (__ENTITY__, 8001,'PCGAFR14-DEV','CAPIT',		'XXXXXX',  	   '10','8000', 'Capital et réserves', 1);
+INSERT INTO llx_accounting_account (entity, rowid, fk_pcg_version, pcg_type, pcg_subtype, account_number, account_parent, label, active) VALUES (__ENTITY__, 8002,'PCGAFR14-DEV','CAPIT',		'XXXXXX', 	  '101','8001', 'Capital', 1);
```

Comment:
@aspangaro Quick question: do you have a script or a trick to generate all these insert lines? Or do you write all these lines manually?